spec/views/show.html.erb_spec.rb

```diff
@@ -1,6 +1,6 @@
-require "rails_helper"
+require 'rails_helper'
 
-describe "home/show" do
+describe 'home/show' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.